logitech_r400/src/logitech_r400/logitech_r400

```diff
@@ -66,9 +63,15 @@ class LogitechR400Usb(object):
         for device in self._devices:
             self.release_device(device)
 
+    def read_config(self):
+        config_file_path = rospkg.RosPack().get_path('logitech_r400') + '/config/config.yaml'
+        with open(config_file_path, 'r') as stream:
+            config = yaml.load(stream)
+        self.product = int('0x' + str(config['product']), 16)
```

Comment:
Please we should stick to our convention of using leading underscore for the member fields.

Reply:
Also, please declare all the class member fields in the constructor.
This is a convention we should always follow.
It makes it possible to see the member fields in a glimpse.
You can initialise them to `None` if the values isn't known yet.